### ISSUE TITLE
ColPack folder structure does not match ColPack.vcxproj

### ISSUE THREAD

**Issue by detzt** [coin-or/ADOL-C#24]
I am currently trying to use ADOL-C under Windows VS2019. During the installation I ran into some problems. One of them is that the folder structure of ColPack does not match the structure set up in the ADOL-C/MSVisualStudio/v14/ColPack.vcxproj file.
Folder names:
- GraphColoring vs. GeneralGraphColoring
- Main vs inc (inc is also in ColPack/inc and not in ColPack/src/main)

Btw. I am not completely sure if I am supposed to place ColPackHeaders.h and Definitions.h in .../v14/ColPack/main or directly in .../v14/ColPack, because both locations were reported as missing these files during building.

**Comment by awalther1** [comment]
Hi,

I checked the combination of ADOL-C and ColPack a few days ago. The difficult point is that the ColPack version in the master branch has a different structure that the release 1.0.10. I contacted the ColPack developers to discuss this issue but did not get an answer so far. In the meantime you could use the following to work with the compressed derivative matices:

Download ColPack version 1.0.10 from

https://github.com/CSCsw/ColPack/releases

and use this version of ColPack. This is what works for me using Linux.
Hope this helps!

Best

Andrea

**Comment by detzt** [comment]
Alright, thank you :)
Considering that the last release is from 2016 it would be the good if they would create a new release anyway.

**Comment by awalther1** [comment]
Yes, this is indeed true. Nevertheless, it would be great if you let me know whether your installation works with the old ColPack version
